viewer-ng/pom.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <!-- would have org.springframework.bootspring-boot-starter-parent</artifactId> as parent, but we need TM as a parent -->
```

Comment:
Possibly dumb question: using the springboot starter as a parent in the tailormap parent module? Maybe for now not possible (everything would get the springboot deps)

Reply:
springboot starte doesn't really do anything apart from 1 plugin; we're using the upstream parent as a BOM in our parent now, so we get all the right versions, but can still override with a latest version eg. Hibernate